clean.sh

```diff
@@ -35,4 +36,4 @@ rm -rf ./ios/Podfile.lock 2>/dev/null
 
 echo "Clearing node modules..."
 
-rm -rf node_modules/ && yarn cache clean && yarn install && npx pod-install && yarn postinstall && code . && clear
\ No newline at end of file
+rm -rf node_modules/ && yarn cache clean && yarn install && npx pod-install && yarn postinstall
```

Comment:
I don't mind removing the clear command but because of line 13, where VSCode is killed, I think I'd prefer to keep the `code .` so that it open back up automatically. If you're not using VSCode, maybe we can pull this out of the continuous script and ignore the command if it errors

Reply:
As as user/developer it was very jarring that vscode would close; I thought it was crashing.

I don't know why the `code .` wasn't working on my macbook air m1; I followed the instructions and installed the cli tool, but it wasn't working.

A middle ground can be found. I lean toward a variable in the .env file that disables the behavior, but by default it stays the way it was.